### PR TITLE
Feature/show slot number update footer

### LINKED ISSUE #387: block endpoint should show slot number
ex. https://explorer.cardano.org/en/block?id=ab8ad351bbe6f8114ccb112a85f942fefc35c824da897f5cc78d2377235d47a2

It gives block number and epoch, but not overall slot number. Overall slot number is arguably the only important piece of information as that's used to set TTLs on transactions, identify how much more your node has to sync, etc... but it doesn't get displayed anywhere in the explorer.

### PR DESCRIPTION
**Pull request by rhyslbw**
### feature: Add global slot number support

The slot in epoch value is less relevant in the post-Byron era,
whereas the global slot number is highly relevant.

- Adds a slot number search.
- Separates the slot number to a new line in epoch summary component

Closes #387

### feature: Add support link to footer

### CODE REVIEW

File: source/features/i18n/translations/de.json [hunk 1]
```diff
@@ -62,6 +64,7 @@
     "epochSummaryTitle": "Epoche Übersicht",
     "lastBlockAt": "Letzter Block",
     "numberOfBlocks": "Anzahl an Blöcken",
+    "numberOfSlots": "Anzahl an slots",
```
Comment (rhyslbw) [inline source/features/i18n/translations/de.json:67]: @DominikGuzei does this need a capital letter? i.e Slots?

File: source/features/i18n/translations/de.json [hunk 2]
```diff
@@ -87,6 +90,7 @@
     "ouroborosAlgorithm": "Ouroboros Algorythmus",
     "project": "Projekt.",
     "shellyExplorer": "Shelley Explorer",
+    "submitASupportRequest": "<<>>",
```
Comment (rhyslbw) [inline source/features/i18n/translations/de.json:93]: @DominikGuzei Please provide translation

File: source/features/i18n/translations/ja.json [hunk 3]
```diff
@@ -15,10 +15,11 @@
     "blockTitle": "ブロック",
     "createdAtTitle": "生成時",
     "createdByTitle": "生成者",
-    "epochSlotTitle": "エポック/スロット",
+    "epochTitle": "エポック",
     "outputTitle": "アウトプット (₳)",
     "pageTitle": "ブロック | $t(productTitle)",
     "sizeTitle": "サイズ（バイト）",
+    "slot": "<<>>",
```
Comment (rhyslbw) [inline source/features/i18n/translations/ja.json:22]: @eddyjlhaigh Please provide translation

File: source/features/i18n/translations/ja.json [hunk 4]
```diff
@@ -128,7 +132,8 @@
     "notExist": "アドレスが見つかりません",
     "notFound": "一致する検索結果はありません",
     "placeholder": "エポック、ブロック、アドレス、トランザクションを検索する",
-    "suggestion_block": "ブロックを検索",
+    "suggestion_block_by_number": "<<>>",
```
Comment (rhyslbw) [inline source/features/i18n/translations/ja.json:135]: @eddyjlhaigh Please provide translations for

- _Search for a block by number_
- _Search for a block by slot number_

File: source/features/i18n/translations/ja.json [hunk 5]
```diff
@@ -62,6 +64,7 @@
     "epochSummaryTitle": "エポック概要",
     "lastBlockAt": "最終ブロック",
     "numberOfBlocks": "ブロック数",
+    "numberOfSlots": "<<>>",
```
Comment (rhyslbw) [inline source/features/i18n/translations/ja.json:67]: @eddyjlhaigh Please provide translation

File: source/features/i18n/translations/ja.json [hunk 6]
```diff
@@ -87,6 +90,7 @@
     "ouroborosAlgorithm": "Ouroborosアルゴリズム",
     "project": "プロジェクトです",
     "shellyExplorer": "Shelleyエクスプローラー",
+    "submitASupportRequest": "<<>>",
```
Comment (rhyslbw) [inline source/features/i18n/translations/ja.json:93]: @eddyjlhaigh Please provide translation

**Comment by DominikGuzei** [comment]
@rhyslbw I will provide the translations! The deploy previews seem to be broken – no data is loaded and I can see these warnings in the console:
grafik

**Pull request merged**